frontend/src/locales/de.json

```diff
@@ -261,5 +281,7 @@
   "transaction-link": {
     "send_you": "sendet dir"
   },
-  "via_link": "über einen Link"
+  "via_link": "über einen Link",
+  "welcome": "Willkommen",
+  "|": " | "
```

Comment:
math.pipe

Reply:
Just put the string `" | "` directly in the code.

If the linting fails just exclude it from linting!
I like it WYSIWYG!!!

Reply:
I understand ... we already have the pipe ... 
I overlooked that... 
I have replaced this